src/components/structures/LoggedInView.tsx

```diff
@@ -748,7 +749,7 @@ class LoggedInView extends React.Component<IProps, IState> {
                                     <LeftPanel
                                         pageType={this.props.page_type as PageTypes}
                                         isMinimized={shouldUseMinimizedUI || false}
-                                        resizeNotifier={this.props.resizeNotifier}
+                                        resizeNotifier={this.context.resizeNotifier}
```

Comment:
ditto for LeftPanel

Reply:
This only gets resizeNotifier so it can pass it into the legacy room list which doesn't seem worth changing at this point, it can just be stripped out when the old room list dies?